lib/runa/client.rb

```diff
@@ -32,10 +30,14 @@ def initialize(options = {})
       end
     end
 
-    def request(method, path, payload = {})
+    def request(method, path, payload = {}, key = '')
       @connection.send(method) do |req|
         req.url [@api_path, path].join
         req.headers['Content-Type'] = 'application/json'
+        req.headers['X-Api-Key'] = @api_key
+        req.headers['X-Api-Version'] = '2024-02-05'
```

Comment:
so any string literal that you don't modify should also the `.freeze` appended. This one and `application/json` above